src/pages/myNotes/noteText/noteText.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import useStyles from '../styled';
+
+const NoteText = ({ text: { title, description, date } }) => {
```

Comment:
It's not a good name for Component. Can you please rename it with something more clear?